cpp/include/cudf/detail/tdigest/tdigest.hpp

```diff
@@ -143,28 +143,29 @@ std::unique_ptr<column> make_tdigest_column(size_type num_rows,
                                             rmm::device_async_resource_ref mr);
 
 /**
- * @brief Create an empty tdigest column.
+ * @brief Create a tdigest column of empty clusters.
  *
- * An empty tdigest column contains a single row of length 0
+ * The column created contains the specified number of rows of empty clusters.
  *
  * @param stream CUDA stream used for device memory operations and kernel launches.
  * @param mr Device memory resource used to allocate the returned column's device memory.
  *
- * @returns An empty tdigest column.
+ * @returns A tdigest column of empty clusters.
  */
 CUDF_EXPORT
-std::unique_ptr<column> make_empty_tdigest_column(rmm::cuda_stream_view stream,
-                                                  rmm::device_async_resource_ref mr);
+std::unique_ptr<column> make_tdigest_column_of_empty_clusters(size_type num_rows,
```

Comment:
I understand respecting the convention is important. However, I'm not sure if that's worth to have the confusing code left unchanged. In terms of maintenance challenges, what kind of challenges do you have in your mind? With Today's advanced IDEs, I'm not sure what challenges I'm missing here.

I'm not quite sure either about how the original name helps with readability. The original name rather seems quite confusing to me. It could be easily read as "making an empty column of the tdigest type" than "making a column of an empty tdigest". Without the renaming, I would likely get confused again if I look at this function in 6 months after I forget what it does.

That being said, I have another proposal for the renaming: `make_empty_tdigests_column`. This seems more explicit to me that what is empty is the tdigests, not the column. What do you think?

Reply:
`make_empty_tdigests_column` is great